.github/workflows/codeql.yml

```diff
@@ -77,7 +71,7 @@ jobs:
     # Command-line programs to run using the OS shell.
     # See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsrun
 
-    #   If the Autobuild fails above, remove it and uncomment the following three lines. 
+    #   If the Autobuild fails above, remove it and uncomment the following three lines.
```

Comment:
why do we need to add codeql just for the onnxruntime repo? The other repos don't need this?

Reply:
CodeQL won't build out-of-the-box for our C++ structure (and it might have some issues with our Java/Go files, though I don't remember). So we only use it for Python. If we have no Python files in a repo, it automatically fails due to not finding any files, so we only have it in the repos that have at least one Python file.